src/main/java/me/zeroeightsix/kami/module/modules/ClickGUI.kt

```diff
@@ -2,18 +2,54 @@ package me.zeroeightsix.kami.module.modules
 
 import me.zeroeightsix.kami.gui.kami.DisplayGuiScreen
 import me.zeroeightsix.kami.module.Module
+import me.zeroeightsix.kami.setting.Setting
+import me.zeroeightsix.kami.setting.Settings
+import me.zeroeightsix.kami.util.TimerUtils
 import org.lwjgl.input.Keyboard
+import kotlin.math.round
 
 /**
  * Created by 086 on 23/08/2017.
- * Updated by Xiaro on 18/08/20
+ * Updated by Xiaro on 28/08/20
  */
 @Module.Info(
         name = "ClickGUI",
         description = "Opens the Click GUI",
-        category = Module.Category.CLIENT
+        category = Module.Category.CLIENT,
+        alwaysListening = true
```

Comment:
why is this `alwaysListening`?

Reply:
So it will update the scale while the click gui is close